src/components/dialogs/filters.js

```diff
@@ -103,9 +109,14 @@ export const RangeSelection = ({ initialValue, onChange, titleMessage }) => {
         setEqualityType(e.target.value);
     }
 
+    const regex = /^[0-9]*[.]?[0-9]*$/;
```

Comment:
In gridstudy, there is this function :
`function isFloatNumber(val) {
    return /^-?[0-9]*[.,]?[0-9]*$/.test(val);
}`
It allows negative numbers and dots/comma instead of only dots.

Reply:
Moreover, because the keyboard blocks the user's input each stroke if the format is not correct, we can have a bad situation for the user :
Step 1 : the user has "0.56111108" as his current value, but wants "0.50111108" instead.
Step 2 : the user copies " 0.50111108" from some place (with a space before, because that's what users do).
Step 3 : the user pastes it in his field, visually **it seems like** it was pasted, but no, the original value is still there.

Reply:
A nominal voltage level is a positive value, then the regexp is ok (I added the "," as you noticed).

to enhance the copy/paste issue you mentionned, we display an info snackbar message in case of bad copy.